js/generate-utils.js

```diff
@@ -208,6 +208,26 @@ const browsers = {
   secondary: ['baidu', 'opera', 'qq', 'samsunginternet', 'uc']
 };
 
+
+/**
+ * Format a date into a month and year string, using the provided locale.
+ *
+ * Rules for Chinese are hardcoded when code runs in JSDOM, because JSDOM does
+ * not yet seem to support producing dates in Chinese.
+ */
+const formatMonthAndYearDate = function (date, lang) {
+  if (((lang === 'zh') || lang.startsWith('zh-')) &&
+      window._runScripts && window._virtualConsole && (window.name === 'nodejs')) {
```

Comment:
I figured I should note that ```window.name === 'nodejs'``` is not supported as a way to detect jsdom, and is planned for removal in the next major version: https://github.com/jsdom/jsdom/pull/2069

While no changes are planned to them at the moment, the properties starting with `_` are also not part of the public API and could change in any version. The recommended way to detect jsdom is to inspect the user agent string instead.

Reply:
Duly noted, and thanks for chiming in, @Zirro! Now tracked in #255.